server/config/toml.go

```diff
@@ -202,7 +201,7 @@ enable-unsafe-cors = {{ .GRPCWeb.EnableUnsafeCORS }}
 [state-sync]
 
 # snapshot-interval specifies the block interval at which local state sync snapshots are
-# taken (0 to disable). Must be a multiple of pruning-keep-every.
+# taken (0 to disable).
```

Comment:
snapshot-interval must be less than or equal to pruning-keep-every if enabled.

Reply:
Que? We're removing `keep-every`. You want me to update the doc here some other way?

Reply:
Oh woops, I meant to write 

snapshot-interval must be less than or equal to pruning-keep-recent if enabled.